lib/ansible/galaxy/data/default/role/meta/main.yml.j2

```diff
@@ -17,22 +20,23 @@ galaxy_info:
   # - CC-BY-4.0
   license: {{ license }}
 
-  min_ansible_version: {{ min_ansible_version }}
-
-  # If this a Container Enabled role, provide the minimum Ansible Container version.
+  # If this a Container Enabled role, provide the minimum Ansible Container
+  # version.
   # min_ansible_container_version:
+  min_ansible_version: "{{ min_ansible_version }}"
 
+  # List tags for your role here, one per line. A tag is a keyword that
+  # describes and categorizes the role. Users find roles by searching for tags.
+  # Be sure to remove the '[]' above, if you add tags to this list.
```

Comment:
These comments provided a visual representation where to add a multi-line list. The last line makes no sense now, same for the dependencies comment below. Also see https://github.com/ansible/ansible/pull/82529.

Reply:
I actually needed to read your comment and the comment in the file twice in order to figure out what was wrong. I've changed the wording slightly to adjust. It should be better now.